h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -197,6 +195,13 @@ public Config setEnableContributions(boolean val) throws IOException {
 
     public boolean getEnableContributions() { return enableContributions; }
 
+    public Config setUseExternalEncoding(boolean val)  {
```

Comment:
We do have JavaDoc on top of most of these methods. Not all, but these are directly user-facing and I'd personally provide it.

Reply:
good point, will add